Tools/WinMLRunner/src/EventTraceHelper.cpp

```diff
@@ -0,0 +1,559 @@
+#include "EventTraceHelper.h"
+#include <direct.h>
+
+#define LOGSESSION_NAME _T("winml")
+
+struct __declspec(uuid("{BCAD6AEE-C08D-4F66-828C-4C43461A033D}")) DXGKRNL_PROVIDER_GUID_HOLDER;

```

Comment:
>DXGKRNL_PROVIDER_GUID_HOLDER [](http://example.com/codeflow?start=66&length=28)

This should be WINML_PROVIDER_GUID_HOLDER right?

Reply:
Yes